setup.py

```diff
@@ -43,14 +36,21 @@
 ]
 PLATFORMS = "Any"
 INSTALL_REQUIRES = ["numpy", "pandas", "xarray", "netCDF4", "packaging"]
+# Configuration for setuptools-scm
+SETUP_REQUIRES = ["setuptools_scm"]
```

Comment:
Do we need to add `setuptools_scm` to requirements-dev.txt and enviroment.yaml?

Reply:
So this is a set-up (i.e. build) requirement (used for packaging to PyPI) rather than a runtime (i.e. end-user) requirement. So I don't think we have to, but the real test will be to see if things get uploaded to TestPyPI properly after this PR is merged.

Reply:
> but the real test will be to see if things get uploaded to TestPyPI properly after this PR is merged.

Perhaps we should enable TestPyPI publishing in this PR to check if it really works as we expect.

Reply:
Yep, upload to TestPyPI works:

![image](https://user-images.githubusercontent.com/23487320/99329620-7de85f00-28e3-11eb-80bf-6b22d0a9f2bb.png)
![image](https://user-images.githubusercontent.com/23487320/99329697-a2443b80-28e3-11eb-8ea1-7e389261fb09.png)

Reply:
Great!

Reply:
Wait. Should it be 0.2.1.dev9? Why 0.2.2.dev9?

Reply:
That's correct right? It should be either 0.2.1.**post**9 (https://www.python.org/dev/peps/pep-0440/#post-releases) or 0.2.2.**dev**9 (https://www.python.org/dev/peps/pep-0440/#developmental-releases).

Reply:
OK. Never read the PEP 440 carefully before.

Reply:
Yeah, there's too many PEPs to read :joy: But I think we're good here.